_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -59,7 +59,32 @@ FIXME:harding to update Tuesday
 [Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
 (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
 
-- [Bitcoin Core #19569][] Enable fetching of orphan parents from wtxid peers FIXME:moneyball
+- [Bitcoin Core #19569][] allows Bitcoin Core to fetch the parents of _orphan_
+  transactions from peers that relay transactions using wtxid. An orphan
+  transaction is an unconfirmed transaction that we receive from a peer where we
+  don't yet have the parent transactions, either as part of our best block
+  chain, or in the mempool. More precisely, an orphan transaction has at least
+  one transaction input whose associated output is not in the UTXO set or our
+  mempool's outpoint map.
+
+    When we receive an orphan transaction, we place it in a temporary data
+    structure called the orphan set. We then ask the peer that sent us the
+    orphan to also send us the parent transactions that we don't yet have. We can
+    do that because the orphan refers to the txids of the parent transactions. We
+    simply send a `GETDATA` message containing those txids to the peer to
```

Comment:
We don't have it in the style guide yet, but so far we've always used lowercase names for P2P messages.  I think this is appropriate because all current message names in the wire format are lowercase, e.g. the header for a getdata message starts with:

```text
f9beb4d9 ................... Start string: Mainnet
676576646174610000000000 ... Command name: "getdata" in ASCII + null padding
```

I know there's a counter argument to use all caps because it's declared as `NetMsgType::GETDATA` in the code, but I note with some amusement that the 2015 [commit](https://github.com/bitcoin/bitcoin/commit/9bbe71b641e2fc985daf127988a14a67c99da50a) which introduced that code convention was not only made after I chose the style of using lowercase names, the commit actually linked to my previously-written documentation using lowercase names.  :-)

Reply:
Great point. I agree that we should standardize on lowercase and include it in our style guide.